tests/functional/preview_and_dev/test_cbc_integration.py

```diff
@@ -309,129 +297,55 @@ def test_broadcast_with_both_azs_failing_eventually_succeeds_if_azs_are_restored
     failure_code = "500"
     success_code = "200"
 
-    try:
-        ddbc = create_ddb_client()
-        _set_response_codes(ddbc, [primary_cbc, secondary_cbc], failure_code)
-
-        broadcast_alert(driver, broadcast_id)
-        (service_id, broadcast_message_id) = _get_service_and_broadcast_ids(
-            driver.current_url
-        )
-        time.sleep(30)  # wait for some retries
-        _set_response_codes(ddbc, [primary_cbc, secondary_cbc], success_code)
-        time.sleep(90)  # wait for more retries
-
-        url = f"/service/{service_id}/broadcast-message/{broadcast_message_id}/provider-messages"
-        response = api_client.get(url=url)
-        assert response is not None
-
-        provider_messages = response["messages"]
-        assert provider_messages is not None
-        assert len(provider_messages) == 4
-
-        request_id = _dict_item_for_key_value(
-            provider_messages, "provider", "three", "id"
-        )
-
-        db_response = ddbc.query(
-            TableName="LoopbackRequests",
-            KeyConditionExpression="RequestId = :RequestId",
-            ExpressionAttributeValues={":RequestId": {"S": request_id}},
-        )
-
-        print(provider_messages)
-        print(db_response)
-
-        responses = db_response["Items"]
-
-        az1_response_codes = _dynamo_items_for_key_value(
-            responses, "MnoName", primary_cbc, "ResponseCode"
-        )
-        print(az1_response_codes)
-
-        az2_response_codes = _dynamo_items_for_key_value(
-            responses, "MnoName", secondary_cbc, "ResponseCode"
-        )
-        print(az2_response_codes)
-
-        response_codes = set(az1_response_codes + az2_response_codes)
-        assert len(response_codes) == 2  # we should have a 200 along with the 500s
-        assert failure_code in response_codes
-        assert success_code in response_codes
-
-    finally:
-        _set_response_codes(ddbc, [primary_cbc, secondary_cbc], success_code)
-        cancel_alert(driver, broadcast_id)
-
-
-# @recordtime
-@pytest.mark.skip(
-    "Celery + SQS interaction on retry needs to be checked before "
-    "this test can be trusted to demonstrate anything useful."
-)
-@pytest.mark.xdist_group(name="broadcasts")
-def test_broadcast_with_both_azs_failing_has_sqs_retry_after_visiblity_timeout(
-    driver, api_client
-):
-    broadcast_id = str(uuid.uuid4())
+    ddbc = create_ddb_client()
+    _set_response_codes(ddbc, "all", "200")
+    _set_response_codes(ddbc, [primary_cbc, secondary_cbc], failure_code)
 
-    primary_cbc = "ee-az1"
-    secondary_cbc = "ee-az2"
-    failure_code = "500"
-    success_code = "200"
+    broadcast_alert(driver, broadcast_id)
+    (service_id, broadcast_message_id) = _get_service_and_broadcast_ids(
+        driver.current_url
+    )
+    time.sleep(10)  # wait for some retries
+    _set_response_codes(ddbc, [primary_cbc, secondary_cbc], success_code)
+    time.sleep(60)  # wait for more retries
```

Comment:
I feel like it would be better to provide a function which checks the state of a broadcast (finished, retrying etc) and just call that periodically rather than wait for a minute or greater?

However I accept that this is better than it was and don't want to hold up progress in favour of perfection.